pkg/provisioner/ironic/testserver/server.go

```diff
@@ -76,6 +76,20 @@ func (m *MockServer) Response(pattern string, payload string) *MockServer {
 	return m.ResponseWithCode(pattern, payload, http.StatusOK)
 }
 
+// SendJSONResponse marshalls the payload to a JSON object and sends
+// the response using the given writer
+func (m *MockServer) SendJSONResponse(payload interface{}, code int, w http.ResponseWriter, r *http.Request) {
```

Comment:
Is this a replacement for [ResponseJSON](https://docs.google.com/document/d/1fzHLNz8bLeu0DbO7f20GazbGiqIWKwHzVKaLLdtTsvo/edit#heading=h.s76i9akrmjw9)?

Reply:
No. ResponseJSON builds a handler with a static response. This formats a response dynamically at the point that it's needed by another handler.

Reply:
I suppose I could have made both of them use the same code at some layer.

Reply:
Agree, looks like there's a good overlap